frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  Form,
+  TextInputTypes,
+  ActionGroup,
+  Alert,
+  Button,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { FormikProps, FormikValues } from 'formik';
+import { ButtonBar } from '@console/internal/components/utils';
+import { MultiColumnField, InputField, DropdownField } from '../formik-fields';
+
+enum accessRoles {
+  '' = 'Select a role',
+  admin = 'Admin',
+  view = 'View',
+}
+
+const ProjectAccess: React.FC<FormikProps<FormikValues>> = ({
+  handleSubmit,
+  handleReset,
+  isSubmitting,
+  status,
+  errors,
+  dirty,
+}) => {
+  return (
+    <Form onSubmit={handleSubmit}>
+      <div className="co-m-pane__form">
+        <MultiColumnField
+          name="projectAccess"
+          addLabel="Add Access"
+          headers={['Name', 'Role']}
+          emptyValues={{ user: '', role: '' }}
+          toolTip="Remove Access"
+        >
+          <InputField name="user" type={TextInputTypes.text} placeholder="Name" />
+          <DropdownField name="role" items={accessRoles} fullWidth />
+        </MultiColumnField>
+        <hr />
+        <ButtonBar
+          inProgress={isSubmitting}
+          errorMessage={status && status.submitError}
+          successMessage={status && status.success}
+        >
+          {!status && dirty && (
```

Comment:
Shouldn't this always be present if the form is dirty? It looks like the logic is copied from another form that may also have an error.
```suggestion
          {dirty && (
```

Reply:
Done